### PR TITLE
Replaced unavailable gcr.io/google_samples image in replicaset.md

### LINKED ISSUE #44415: Image `gcr.io/google_samples/gb-frontend:v3` unavailable in ReplicaSet example
**This is a Bug Report**

**Problem:**
The container image (gcr.io/google_samples/gb-frontend:v3) used in the first example on the page https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/ is not working.

**Proposed Solution:**
replace it with a working image 

**Page to Update:**

https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/

image2
image

### PR DESCRIPTION
**Pull request by aj11anuj**
This PR fixes #44415 
Refer PR #44290 for validating correctness of Image.

### CODE REVIEW

**Comment by mengjiao-liu** [comment]
I run the following command and see that there are a few similar images that need to be updated as well(Blogs that are too old should not be updated), maybe we can update them all in this PR?
```
$ grep -rn "gcr.io/google_samples"  ./content/en
./content/en/docs/tasks/configure-pod-container/translate-compose-kubernetes.md:106:       image: gcr.io/google_samples/gb-redisslave:v3
./content/en/docs/concepts/workloads/controllers/replicaset.md:103:    Image:        gcr.io/google_samples/gb-frontend:v3
./content/en/blog/_posts/2015-12-00-Managing-Kubernetes-Pods-Services-And-Replication-Controllers-With-Puppet.md:52:  redis_slave_image => 'gcr.io/google_samples/gb-redisslave:v1',
./content/en/blog/_posts/2015-12-00-Managing-Kubernetes-Pods-Services-And-Replication-Controllers-With-Puppet.md:53:  frontend_image => 'gcr.io/google_samples/gb-frontend:v3',     
./content/en/examples/controllers/frontend.yaml:21:        image: gcr.io/google_samples/gb-frontend:v3

```

**Comment by aj11anuj** [comment]
Sure 👍

File: content/en/docs/concepts/workloads/controllers/replicaset.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Pod Template:
   Labels:  tier=frontend
   Containers:
    php-redis:
-    Image:        gcr.io/google_samples/gb-frontend:v3
+    Image:        us-docker.pkg.dev/google-samples/containers/gke/gb-frontend:v3
```
Comment (kundan2707) [inline content/en/docs/concepts/workloads/controllers/replicaset.md:103]:
```suggestion
    Image:        us-docker.pkg.dev/google-samples/containers/gke/gb-frontend:v5
```

File: content/en/examples/controllers/frontend.yaml [hunk 2]
```diff
@@ -18,4 +18,4 @@ spec:
     spec:
       containers:
       - name: php-redis
-        image: gcr.io/google_samples/gb-frontend:v3
+        image: us-docker.pkg.dev/google-samples/containers/gke/gb-frontend:v3
```
Comment (kundan2707) [inline content/en/examples/controllers/frontend.yaml:21]:
```suggestion
        image: us-docker.pkg.dev/google-samples/containers/gke/gb-frontend:v5
```

File: content/en/docs/concepts/workloads/controllers/replicaset.md (hunk 1 above)
Comment (kundan2707) [reply content/en/docs/concepts/workloads/controllers/replicaset.md:103]: Only one tag v5 exists and tested v3 will also not work as not available in repository.

File: content/en/examples/controllers/frontend.yaml (hunk 2 above)
Comment (kundan2707) [reply content/en/examples/controllers/frontend.yaml:21]: Only one tag v5 exists and tested v3 will also not work as not available in repository.

**Comment by kundan2707** [comment]
squash commits after resolving comments.

**Review by kundan2707 — COMMENTED** [review]
I tested by self too.
Thanks 
/lgtm

**Comment by aj11anuj** [comment]
😓

**Pull request closed without merging**